pkgs/applications/altcoins/spruned/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, pythonPackages }:
+
+with stdenv.lib;
+pythonPackages.buildPythonApplication rec {
+  pname = "spruned";
+  name = "${pname}-${version}";
```

Comment:
this line is unneeded because it is automatically done by `buildPythonApplication`